parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestEncryptionOptions.java

```diff
@@ -0,0 +1,660 @@
+          ColumnEncryptionProperties columnProperties50 = ColumnEncryptionProperties
+            .builder(DOUBLE_FIELD_NAME)
+            .withKey(COLUMN_ENCRYPTION_KEY1)
+            .withKeyID(COLUMN_ENCRYPTION_KEY1_ID)
+            .build();
+
+          ColumnEncryptionProperties columnProperties51 = ColumnEncryptionProperties
+            .builder(FLOAT_FIELD_NAME)
+            .withKey(COLUMN_ENCRYPTION_KEY2)
+            .withKeyID(COLUMN_ENCRYPTION_KEY2_ID)
+            .build();
+          columnPropertiesMap5.put(columnProperties50.getPath(), columnProperties50);
+          columnPropertiesMap5.put(columnProperties51.getPath(), columnProperties51);
+
+          encryptionPropertiesMap.put(EncryptionConfiguration.ENCRYPT_COLUMNS_AND_FOOTER_DISABLE_AAD_STORAGE,
+            FileEncryptionProperties.builder(FOOTER_ENCRYPTION_KEY)
+              .withFooterKeyMetadata(footerKeyMetadata)
+              .withEncryptedColumns(columnPropertiesMap5)
+              .withAADPrefix(AADPrefix)
+              .withoutAADPrefixStorage()
+              .build());
+          break;
+
+        case ENCRYPT_COLUMNS_AND_FOOTER_CTR:
+          // Encryption configuration 6: Encrypt two columns and the footer, with different keys.
+          // Use AES_GCM_CTR_V1 algorithm.
+          Map<ColumnPath, ColumnEncryptionProperties> columnPropertiesMap6 = new HashMap<>();
+          ColumnEncryptionProperties columnProperties60 = ColumnEncryptionProperties
+            .builder(DOUBLE_FIELD_NAME)
+            .withKey(COLUMN_ENCRYPTION_KEY1)
+            .withKeyID(COLUMN_ENCRYPTION_KEY1_ID)
+            .build();
+
+          ColumnEncryptionProperties columnProperties61 = ColumnEncryptionProperties
+            .builder(FLOAT_FIELD_NAME)
+            .withKey(COLUMN_ENCRYPTION_KEY2)
+            .withKeyID(COLUMN_ENCRYPTION_KEY2_ID)
+            .build();
+          columnPropertiesMap6.put(columnProperties60.getPath(), columnProperties60);
+          columnPropertiesMap6.put(columnProperties61.getPath(), columnProperties61);
+
+          encryptionPropertiesMap.put(EncryptionConfiguration.ENCRYPT_COLUMNS_AND_FOOTER_CTR,
+            FileEncryptionProperties.builder(FOOTER_ENCRYPTION_KEY)
+              .withFooterKeyMetadata(footerKeyMetadata)
+              .withEncryptedColumns(columnPropertiesMap6)
+              .withAlgorithm(ParquetCipher.AES_GCM_CTR_V1)
+              .build());
+          break;
+
+        case NO_ENCRYPTION:
+          // Encryption configuration 7: Do not encrypt anything
+          encryptionPropertiesMap.put(EncryptionConfiguration.NO_ENCRYPTION, null);
+          break;
+      }
+    }
+    return encryptionPropertiesMap;
+  }
+
+  /**
+   * Create a number of Decryption configurations
+   * @param AADPrefix
+   * @return
+   */
+  private Map<DecryptionConfiguration, FileDecryptionProperties>  getDecryptionConfigurations(byte[] AADPrefix) {
```

Comment:
Similarly to the previous comment, you may add these to the `DecryptionConfiguration` enum.